src/components/DocumentationTopic/BetaLegalText.vue

```diff
@@ -46,7 +46,7 @@ export default {
   background-color: var(--color-fill-secondary);
 
   &-container {
-    @include breakpoint-content;
+    @include breakpoint-dynamic-sidebar-content;
```

Comment:
Use this, in case someone ends up in an archive without a sidebar

```suggestion
    @include dynamic-content-container;
```